rfc/rfc-51/rfc-51.md

```diff
@@ -42,11 +43,11 @@ In cases where Hudi tables used as streaming sources, we want to be aware of all
 
 To implement this feature, we need to implement the logic on the write and read path to let Hudi figure out the changed data when read. In some cases, we need to write extra data to help optimize CDC queries.
 
-## Scenarios
+## Scenario Illustration
```

Comment:
Can we call out / illustrate the scenarios where insert&delete and delete&insert should produce separate CDC rows and that is a requirement for this design

Reply:
> should produce separate CDC rows

I guess this is a must ? How could you combine them in one row, the schema does not match.

Reply:
I am just calling out that insert and delete should not skip the row key in the CDC stream. I see a implementation of producing a CDC stream with just from and to version and this does not cover the above requirement. Calling this out explicitly helps.

Reply:
updated